GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -67,11 +69,14 @@ public RevisionFileTreeControl()
             _revisionFileTreeController = new RevisionFileTreeController(() => Module.WorkingDir,
                                                                          new GitRevisionInfoProvider(() => Module),
                                                                          new FileAssociatedIconProvider());
+            blameControl.HideCommitInfo();
         }
 
-        public void Bind(Action refreshGitStatus)
+        public void Bind(RevisionGridControl? revisionGrid, RevisionFileTreeControl revisionFileTree, Action refreshGitStatus)
         {
+            _revisionGrid = revisionGrid;
             _refreshGitStatus = refreshGitStatus;
+            _revisionFileTree = revisionFileTree;
```

Comment:
Same sequence as arguments❔
```suggestion
            _revisionFileTree = revisionFileTree;
            _refreshGitStatus = refreshGitStatus;
```

Reply:
ok